contracts/lupi.sol

```diff
@@ -15,17 +20,35 @@ contract lupi is owned {
     State state = State.Betting;
 
     Ticket[] tickets;
-    uint revealedCount;
+    uint revealedCount; // TODO: shall we replace this with a getter returning revealedBets.length?
 
     // bet => tickets
     mapping(uint => uint[]) revealedTickets;
     uint[] revealedBets;
 
     uint winningTicket;
 
-    function lupi() {
+    // TODO: function getBets(address _player) constant returns bets[]
+
+    function lupi(uint _requiredBetAmount, uint _ticketCountLimit, uint _feePt ) {
```

Comment:
It's definitely a good idea to validate arguments. Require would work the same way here, throws.